content/master/get-started/get-started-with-mrds.md

```diff
@@ -0,0 +1,529 @@
+* EC2 instances (exact match)
+* S3 buckets (exact match)  
+* All RDS resources (wildcard match)
+
+## Verify activation
+
+Check that the specified MRDs are now active:
+
+```shell
+kubectl get mrds instances.ec2.aws.crossplane.io buckets.s3.aws.crossplane.io
+```
+
+```shell
+NAME                              STATE    AGE
+instances.ec2.aws.crossplane.io   Active   5m
+buckets.s3.aws.crossplane.io     Active   5m
+```
+
+List all RDS MRDs to see they're also active:
+
+```shell
+kubectl get mrds | grep rds
+```
+
+```shell
+NAME                                      STATE    AGE
+clusters.rds.aws.crossplane.io           Active   5m
+databases.rds.aws.crossplane.io          Active   5m
+dbinstances.rds.aws.crossplane.io        Active   5m
+# ... other RDS resources
+```
+
+## Verify custom resource definition creation
+
+Now that MRDs are active, CRDs exist:
+
+```shell
+kubectl get crds | grep -E "(instances.ec2|buckets.s3|rds)" | head -5
+```
+
+```shell
+buckets.s3.aws.crossplane.io                    2024-01-15T10:30:00Z
+clusters.rds.aws.crossplane.io                  2024-01-15T10:30:00Z
+databases.rds.aws.crossplane.io                 2024-01-15T10:30:00Z
+dbinstances.rds.aws.crossplane.io               2024-01-15T10:30:00Z
+instances.ec2.aws.crossplane.io                 2024-01-15T10:30:00Z
+```
+
+## Create a managed resource
+
+Now you can create managed resources using the active MRDs. Create an S3 bucket:
+
+```yaml
+apiVersion: s3.aws.crossplane.io/v1alpha1
+kind: Bucket
+metadata:
+  name: my-demo-bucket
+spec:
+  forProvider:
+    region: us-east-1
+  providerConfigRef:
+    name: default
+```
+
+{{< hint "note" >}}
+This example assumes you have AWS credentials configured. See 
+[ProviderConfig documentation]({{< ref "../managed-resources/managed-resources#providerconfigref" >}}) for 
+authentication setup.
+{{< /hint >}}
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: s3.aws.crossplane.io/v1alpha1
+kind: Bucket
+metadata:
+  name: my-demo-bucket
+spec:
+  forProvider:
+    region: us-east-1
+  providerConfigRef:
+    name: default
+EOF
+```
+
+## Test inactive resources
+
+Try to create a managed resource for an inactive MRD, like EKS clusters:
+
+```shell
+kubectl get mrd clusters.eks.aws.crossplane.io
+```
+
+```shell
+NAME                              STATE      AGE
+clusters.eks.aws.crossplane.io   Inactive   8m
+```
+
+Attempting to create an EKS cluster fails because the CRD doesn't exist:
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: eks.aws.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: test-cluster
+spec:
+  forProvider:
+    region: us-east-1
+    version: "1.21"
+EOF
+```
+
+```shell
+error validating data: ValidationError(Cluster): unknown field "apiVersion" in io.k8s.api.core.v1.Cluster
+```
+
+## Expand activation with wildcards
+
+Add more resources using wildcard patterns. Update your activation policy:
+
+```shell
+kubectl patch mrap aws-demo-resources --type merge -p '{
```

Comment:
nit: I prefer `kubectl apply -f - <<EOF` with full manifests, so that people can copy paste from any point in the guide, rather than incremental patches. Definitely not a showstopper though.

Reply:
I believe the other getting started guides _don't_ currently use the EOF style. This guide should match them closely stylistically.

FWIW preferences like this in our docs have been a big challenge over the years. We've gone back and forth on here docs. Last time we talked about it I believe the consensus was to do what the Kubernetes docs do, which is:

1. Example manifests are distinct files in the docs repo
2. Our docs site tool (currently Hugo) embeds those files inline
3. We include `kubectl apply -f https://docs.crossplane.io/...` links to quickly apply them

IMO the Kube docs approach gives the best of both worlds.

* You can quickly apply a manifest by copying a CLI command
* The example manifest shown inline is plain old YAML, not YAML with arcane shell stuff around it

Reply:
I don't think we had a tracking issue for this - raised https://github.com/crossplane/docs/issues/960.